var/spack/repos/builtin/packages/tblite/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Tblite(MesonPackage):
+    """Light-weight tight-binding framework"""
+
+    homepage = "https://tblite.readthedocs.io"
+    url = "https://github.com/tblite/tblite/releases/download/v0.3.0/tblite-0.3.0.tar.xz"
+    git = "https://github.com/tblite/tblite.git"
+
+    maintainers = ["awvwgk"]
```

Comment:
The syntax for adding maintainers has changed.  See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers

```suggestion
    maintainers("awvwgk")
```